ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/.content.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    cq:icon="panel"
+    cq:isContainer="{Boolean}true"
+    jcr:description="Displays content as an accordion"
+    jcr:primaryType="cq:Component"
+    jcr:title="Accordion (v1)"
+    sling:resourceSuperType="core/fd/components/form/panelcontainer/v1/panelcontainer"
```

Comment:
should be sites accordion

Reply:
Shouldn't we extend panelcontainer to inherit cq_dialog? Else we will have to repeat cq-dialog from panelcontainer.

Reply:
yes you will need to add dialog components like granite include, similar pattern is followed in panelContainer dialog, which includes items from base component dialog.
you need to extend from tab

Reply:
changed to sites accordion, added include paths from panelcontainer dialog for forms specific settings.